### DISCUSSION TITLE
Recognition of digital numbers wrong

### DISCUSSION THREAD

**Discussion by tk5011** [jomjol/AI-on-the-edge-device#2739] [General]
Hi all,
first of all I wanted to say a huge THANKS for this great project.

I had it up and running now for several months and it works like a charm. However since 2 weeks, I have daily issues with the recognition of one particular digital number. The number states clearly 0, however it is interpreted as 3.
I didn't find any way yet how to correct this. 

To somehow have some values being sent to Home Assistant, I regularly update the "Previous Value".

What is the reason for the false detection of the number? It would be great, if you could help me.

<img width="1029" alt="Bildschirm­foto 2023-12-11 um 22 14 08" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/73420901/0edf53a2-62c8-4bc0-b708-33debb9fbe24">
<img width="1029" alt="Bildschirm­foto 2023-12-11 um 22 14 47" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/73420901/df506d60-a568-4c82-8206-3a90d21d45a5">
<img width="1029" alt="Bildschirm­foto 2023-12-11 um 22 14 29" src="https://github.com/jomjol/AI-on-the-edge-device-docs/assets/73420901/f79e6dcc-4bf1-4764-b9f6-cdc8e89580f5">

**Comment by ghost** [comment]
Try to reduce the reflection. Mostly a dark tape helps (under the digits). You use dig-cont. You can try the dig-class100.

The ROIs could be 1-2 pixels wider.
